src/Microsoft.ComponentDetection/Program.cs

```diff
@@ -18,8 +22,32 @@
         }
     }
 
+    var providers = new LoggerProviderCollection();
```

Comment:
This `LoggerProviderCollection` may be unnecessary since the CG task isn't directly loading Component Detection into its AppDomain (We just register our services inside their `ServiceCollection`). This will simplify our setup tremendously.